### ISSUE TITLE
Drop color not working

### ISSUE THREAD

**Issue by saihaj** [ShabadOS/theme-tool#16]
CSS issue
> Drop color doesn't seem to work

_Originally posted by @bhajneet in https://github.com/ShabadOS/theme-tool/pull/1#issuecomment-623015497_

**Comment by bhajneet** [comment]
On hold from #26 

Basically you cannot use full rgba(n, n, n, n) format. As per the docs you need to capture the string "n, n, n" without opacity.

Compare:

Definition
![image](https://user-images.githubusercontent.com/14130567/82737303-2e432500-9cfe-11ea-8877-a226bed9bebe.png)

Variable
![image](https://user-images.githubusercontent.com/14130567/82737313-44e97c00-9cfe-11ea-867d-d259ea4eb1fb.png)

Variable needs to be only "R, G, B" as a string to fit in the CSS definition. Done this way to avoid user from entering opacity. If you can disable, then we can update desktop overlay to use full string probably. Either way works

**Comment by saihaj** [comment]
Once #28 is approved what is the next step @bhajneet ?

**Comment by bhajneet** [comment]
I was thinking we have two options, however one of them is breaking.

The non breaking change would be to respect the current requirements for
theme files which would be an r, g, b string.

See example.css in the overlay folder of Shabad OS for the comments

**Comment by bhajneet** [comment]
For further clarification, as you can see here:

![image](https://user-images.githubusercontent.com/14130567/82737303-2e432500-9cfe-11ea-8877-a226bed9bebe.png)

The var(--overlay-primary-drop-color) needs to be `R, G, B` to fit in the above picture

So for example a black drop color would be `0, 0, 0`
